node/pkg/driver/node_utils.go

```diff
@@ -210,12 +216,14 @@ func (n NodeUtils) GetSysDevicesFromMpath(device string) (string, error) {
 	}
 
 	logger.Debugf("found slaves : {%v}", slaves)
-	slavesString := ""
+
+	var slavesNames []string
```

Comment:
Please replace slave with secondary

Reply:
as [mentioned here before](https://github.com/IBM/ibm-block-csi-driver/pull/255#discussion_r522051767), this is not part of this PR

Reply:
Please open a ticket to do so (this is an issue which need to be address)

Reply:
already open (see link)